ethereum/core/src/test/java/org/hyperledger/besu/ethereum/transaction/TransactionSimulatorTest.java

```diff
@@ -518,18 +518,19 @@ public void shouldReturnSuccessfulResultWhenEip1559TransactionProcessingIsSucces
             .payload(callParameter.getPayload())
             .signature(FAKE_SIGNATURE)
             .build();
-    mockProcessorStatusForTransaction(expectedTransaction, Status.SUCCESSFUL);
+    mockProcessorStatusForTransaction(expectedTransaction, Status.FAILED);
 
     final Optional<TransactionSimulatorResult> result =
         transactionSimulator.process(callParameter, 1L);
 
-    assertThat(result.get().isSuccessful()).isTrue();
+    assertThat(result.get().isSuccessful()).isFalse();
```

Comment:
Why has this seemingly unrelated test been changed to a failure case? The name still suggests it should be successful too.

Reply:
The actual result doesn't matter for this test in fact they are all mocked base on the param you pass in the mockProcessorStatusForTransaction method. I've refactored the tests a little bit to make it less dependent on this mock.

Reply:
I realize now I touched the wrong test, fixed it to be as it was originally.